__tests__/app.test.js

```diff
@@ -10,6 +10,51 @@ afterAll(() => {
     connection.end()
 })
 
+describe.only('GET /api', () => {
+    test('get status 200 response', () => {
+        return request(app).get('/api').expect(200);
+    });
+    test('returns object containing all of the available endpoints of the api', () => {
+        return request(app).get('/api').expect(200).then((response)=> {
+            expect(response.body.endpoints).toEqual({
```

Comment:
Good test here 😄 Just a suggestion but this test can be condensed by requiring in the endpoints.json at the top of your tests and writing this instead:
 **expect(response.body.endpoints).toEqual(endpointsJson);** let me know if you need more info about this!